cmd/swarm-smoke/upload_and_sync.go

```diff
@@ -338,6 +369,29 @@ func uploadAndSync(c *cli.Context, randomBytes []byte) error {
 	return nil
 }
 
+func isPushSynced(wsHost string, tagname string) (bool, error) {
+	rpcClient, err := rpc.Dial(wsHost)
+	if rpcClient != nil {
+		defer rpcClient.Close()
+	}
+
+	if err != nil {
+		log.Error("error dialing host", "err", err)
+		return false, err
+	}
+
+	var isSynced bool
+	err = rpcClient.Call(&isSynced, "bzz_isPushSynced", tagname)
```

Comment:
This API is still not merged to `master`, but since it is behind a feature flag, I think it is fine to get the tests part first.